db/migrate/20161219162030_add_index_into_app.rb

```diff
@@ -0,0 +1,10 @@
+require 'mongoid'
```

Comment:
We can probably skip this migration. This should be covered in the upgrade instructions:
https://github.com/errbit/errbit#upgrading